finn-no/FinniversKit
# Replace carthage for SwiftPM for swift-snapshot-testing + Xcode 11 updates

# Why?

- SwiftPM is more flexible than carthage (ex: to use development version of packages) and has native support in Xcode 11

# What?

- Remove carthage
- Reinstall swift-snapshot-test with SPM
- Update circleCI settings for Xcode 11
- Update reference snapshots for iOS 13 + iPhone 11 Pro: ⚠️ if you see the image changes in GithHub, you can notice some updates regarding spacing, fonts coming from iOS 13, the same as the new appearance for the `UISegmentedControl`, which I will fix in a new PR after.

# Show me

| Before | After |
| --- | --- |
| ![before](https://raw.githubusercontent.com/finn-no/FinniversKit/861bb815f8da622d53c42aa9243fe772660d1550/UnitTests/__Snapshots__/FullscreenViewTests/testAddressView.iPhone.png) | ![after](https://raw.githubusercontent.com/finn-no/FinniversKit/b3b225b38ba8c35e832252a5912c13c66d55d490/UnitTests/__Snapshots__/FullscreenViewTests/testAddressView.iPhone.png)
 |
